scipy/_lib/_util.py

```diff
@@ -327,7 +328,7 @@ def getfullargspec_no_self(func):
     defaults = tuple(
         p.default for p in sig.parameters.values()
         if (p.kind == inspect.Parameter.POSITIONAL_OR_KEYWORD and
-           p.default is not p.empty)
+            p.default is not p.empty)
```

Comment:
these are (unrelated) pep8 cleanups, right?

Reply:
Yes.  Since there were just a couple issues found by `flake8`, I included the PEP-8 fixes along with my other changes.

Reply:
ok, they look fine to me, just wanted to confirm my understand of why the changes were in the PR.